src/sonic-yang-models/yang-models/sonic-device_metadata.yang

```diff
@@ -120,6 +120,14 @@ module sonic-device_metadata {
                     }
                     default enable;
                 }
+
+                leaf cloudtype {
```

Comment:
Better to add 1 +ve test in https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-yang-models/tests/yang_model_tests/tests_config/device_metadata.json. So that yang-model PKg also load the config via Libyang.

Reply:
@praveen-li could you please review again?